lib/mobile_app_backend/health/checker/alerts.ex

```diff
@@ -11,25 +11,50 @@ defmodule MobileAppBackend.Health.Checker.Alerts.Impl do
 
   alias MBTAV3API.Repository
   alias MBTAV3API.Store
+  alias MobileAppBackend.Health.Checker.Alerts.LastFreshStore
+
+  require Logger
 
   @behaviour MobileAppBackend.Health.Checker
 
+  @grace_period 300
+
   @impl true
   def check_health do
-    store_count = length(Store.Alerts.fetch([]))
+    store_count = length(Store.Alerts.fetch(fields: [alert: []]))
```

Comment:
We sometimes have seen timeouts on health checks ([ticket](https://app.asana.com/1/15492006741476/project/1205425564113216/task/1209680445093527?focus=true)). My hypothesis is that since we don't actually need the alert fields here, requesting the sparse data will be faster.